src/DotVVM.Samples.Common/ViewModels/FeatureSamples/Validation/NullValidationTargetViewModel.cs

```diff
@@ -11,12 +11,12 @@ namespace DotVVM.Samples.BasicSamples.ViewModels.FeatureSamples.Validation
     public class NullValidationTargetViewModel : DotvvmViewModelBase
     {
         [Required]
-        public SomeValidatebleObject NullObject { get; set; }
+        public SomeValidatableObject NullObject { get; set; }
         [Required]
-        public SomeValidatebleObject RealObject { get; set; } = new SomeValidatebleObject();
+        public SomeValidatableObject RealObject { get; set; } = new SomeValidatebleObject();
```

Comment:
This curently breaks build.

```suggestion
        public SomeValidatableObject RealObject { get; set; } = new SomeValidatableObject();
```